### PR TITLE
Karkakol/update version

### PR DESCRIPTION
**Pull request by karkakol**

### CODE REVIEW

File: example/android/build.gradle [hunk 1]
```diff
@@ -21,8 +21,11 @@ buildscript {
     dependencies {
         classpath("com.android.tools.build:gradle")
         classpath('com.facebook.react:react-native-gradle-plugin')
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin")
+//       classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
```
Comment (graszka22) [inline example/android/build.gradle:25]: ?

File: example/ios/Podfile [hunk 2]
```diff
@@ -66,7 +66,7 @@ target 'reactnativemembranewebrtcexample' do
       # necessary for Mac Catalyst builds
       :mac_catalyst_enabled => false
     )
-    __apply_Xcode_12_5_M1_post_install_workaround(installer)
+#     __apply_Xcode_12_5_M1_post_install_workaround(installer)
```
Comment (graszka22) [inline example/ios/Podfile:69]: remove comments

File: package.json [hunk 3]
```diff
@@ -38,35 +38,40 @@
...
+    "expo": "^50.0.7",
     "phoenix": "^1.7.6",
     "promise-fs": "^2.1.1"
   },
   "devDependencies": {
+    "@babel/core": "7.23.9",
+    "@babel/eslint-parser": "7.23.10",
+    "@babel/plugin-proposal-private-property-in-object": "7.21.11",
+    "@babel/preset-env": "7.23.9",
+    "@react-native/babel-preset": "0.73.21",
```
Comment (graszka22) [inline package.json:50]: do we need all of this? shouldn't those packages be added by other packages?

File: example/android/build.gradle (hunk 1 above)
Comment (karkakol) [reply example/android/build.gradle:25]: deleted

File: example/babel.config.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ const path = require('path');
 module.exports = function (api) {
   api.cache(true);
   return {
-    presets: ['babel-preset-expo'],
+    presets: ['module:@react-native/babel-preset'],
```
[the remark this answers is not in the document]
Comment (karkakol) [reply example/babel.config.js:5]: because of upgrading RN to 0.73

File: package.json (hunk 3 above)
Comment (karkakol) [reply package.json:50]: removed

**Pull request merged**